### DISCUSSION TITLE
Questions and Answers blueprint

### DISCUSSION THREAD

**Discussion by notflip** [statamic/cms#7022] [Q&A] [answered]
I'm building a slideshow which has 5 questions, and each question has 1, 2 or 3 possible answers, of which one answer is correct.

I'm fairly new to Statamic, and I'm wondering what the best method for building this would be

- Each question is a Taxonomy (survey_entries), and each answer to the question is a nested taxonomy?
- Each question is a Taxonomy (survey_entries), and I add 3 text fields for possible options

Any other ideas on how I could solve this? Thank you!

**Comment by ryanmitchell** [comment, marked as answer]
I would build an entries collection rather than a taxonomy, and each answer is a row in a grid or replicator field.

**Reply by notflip** [reply]
Doesn't an Entries Collection need a taxonomy? You have to define a taxonomy for each entry collection, no?

**Reply by jasonvarga** [reply]
You don't need a taxonomy until you need one. 😄 There's no requirement.
